README.md

```diff
@@ -18,12 +18,16 @@ In order to get started with the project we will need to do the following:
 
 ### Summary
 
-In this step we will create a package.json file to keep track of our dependencies and their versions for this project.
+In this step we will create a package.json file to keep track of our dependencies and their versions for this project and also create a `.gitignore` file so our `node_modules` don't get tracked by GitHub. Before we get started, it's important to understand the significance of what the `package.json` does for us. For example, if you have a large corporation working on the same project and one person decides to add new packages to the repository and doesn't tell everyone, the code base isn't going to work for everyone else. Rather than having to remember to send a notification to everyone and everyone manually installing those packages wouldn't it be much more simple if that information was already tracked and they could install those packages with one command? Also what if a specific version number of a package needs to be tracked as well? Having a package.json solves these issues. Whenever some one adds a package to a project using `npm install` they can add a `--save` or a `--save --save-exact` tag so it will automatically be added to the package.json. And whenever someone else needs those new packages they can just run `npm install` to get the packages they are missing and also know they are getting the correct versions of those packages.
```

Comment:
Maybe line breaks would help here? Could break it up a little bit so it doesn't feel like an essay. Tim will have ideas too.